### PR TITLE
feat(tianmu): Improve the readbility of stonedb. (#11)

### PR DESCRIPTION
**Pull request by konghaiya**
[summary]
    1 class member variable refactor：exproter directory;
    2 class member variable refactor：loader directory;

## Summary about this PR

Issue Number: close #issue_number_you_created

## Tests Check List

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog

- [ ] New Feature
- [ ] Bug Fix
- [x] Improvement
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

**Review by hustjieke — APPROVED** [review]
LGTM

File: storage/tianmu/exporter/data_exporter.h [hunk 1]
```diff
@@ -46,22 +46,22 @@ class DataExporter {
   virtual void PutRowEnd() = 0;
 
  protected:
-  int cur_attr;
-  std::vector<core::AttributeTypeInfo> deas;
-  std::vector<core::AttributeTypeInfo> source_deas;
-  fields_t _fields;
+  int cur_attr_;
```
Comment (RingsC) [inline storage/tianmu/exporter/data_exporter.h]: You should keep in mind that: 
readability is not just adding some underscore, but let other to know what you want to do just only via your code.

Therefore,` deas_` means what ? I think `attr_infos_` or something else is a better choice.

File: storage/tianmu/exporter/data_exporter.h [hunk 2]
```diff
@@ -46,22 +46,22 @@ class DataExporter {
...
+  std::vector<core::AttributeTypeInfo> deas_;
+  std::vector<core::AttributeTypeInfo> source_deas_;
+  fields_t fields_;
 
-  std::shared_ptr<system::LargeBuffer> buf;
-  system::ChannelOut *progressout;
+  std::shared_ptr<system::LargeBuffer> buf_;
+  system::ChannelOut *progressout_;
```
Comment (RingsC) [inline storage/tianmu/exporter/data_exporter.h]: all same things.

**Review by RingsC — CHANGES_REQUESTED** [review]

File: storage/tianmu/exporter/data_exporter.h (hunk 1 above)
Comment (konghaiya) [reply storage/tianmu/exporter/data_exporter.h]: Some obscure names have been adjusted

**Review by RingsC — APPROVED** [review]

**Pull request closed without merging**